build.gradle

```diff
@@ -161,6 +161,7 @@ android {
 		disable 'GradleDependency' // TODO update to latest support-v4 lib and re-enable this rule
 
 		warningsAsErrors true
+		// abortOnError false
```

Comment:
... which can be temporarily disabled by uncommenting this line, which is why I left it in (for future convenience)